src/elements/content-sidebar/versions/StaticVersionSidebar.js

```diff
@@ -0,0 +1,96 @@
+/**
+ * @flow
+ * @file Versions Sidebar component
+ * @author Box
+ */
+
+import * as React from 'react';
+import { FormattedMessage } from 'react-intl';
+
+import BoxDrive140 from '../../../illustration/BoxDrive140';
+
+import { BackButton } from '../../common/nav-button';
+import Button from '../../../components/button';
+import { LoadingIndicatorWrapper } from '../../../components/loading-indicator';
+import VersionsMenu from './VersionsMenu';
+
+import messages from './messages';
+import './StaticVersionsSidebar.scss';
+
+type Props = {
+    isLoading: boolean,
+    onUpgradeClick: () => void,
+    parentName: string,
+    showUpsellWithPicture: boolean,
+};
+
+const StaticVersionsSidebar = ({ isLoading, onUpgradeClick, parentName, showUpsellWithPicture }: Props): React.Node => {
+    const versionTimestamp = new Date();
+    versionTimestamp.setDate(versionTimestamp.getDate() - 1);
+
+    const versions = ['1', '2', '3'].map(versionNumber => {
+        return {
+            id: versionNumber,
+            version_number: versionNumber,
+            type: 'file_version',
+            permissions: {
+                can_preview: true,
+            },
+            created_at: versionTimestamp.toUTCString(),
+            modified_by: null,
+            size: 1875887,
+            trashed_at: null,
+            uploader_display_name: messages.versionStaticUser.defaultMessage,
```

Comment:
Do we need to internationalize the name of the uploader? If not, let's hardcode it here. If so, we'll need to bring in `intl.formatMessage` rather than referencing the default message, which is always in English.

Reply:
ah, good point, we dont need to translate it, so I will switch to hardcoding it.